src/autotracker.cc

```diff
@@ -9,25 +9,33 @@
 
 namespace toggl {
 
+static const char kTermSeparator = '\t';
+
 bool AutotrackerRule::Matches(const TimelineEvent &event) const {
-    if (Poco::UTF8::toLower(event.Filename()).find(term_)
```

Comment:
Where did filename go?

Reply:
Currently, we don't match by filename anyway - `TimelineEvent` isn't populated with the filename:
https://github.com/toggl-open-source/toggldesktop/blob/ce67d39051d9d55594384a2b25931aa39152163c/src/window_change_recorder.cc#L79-L83
So I removed it for now to avoid confusion. We can bring it back in a separate PR, double-checking if the UI and support article informs the user of this capability.

Reply:
:+1: